packages/@react-aria/select/src/useSelect.ts

```diff
@@ -182,6 +189,9 @@ export function useSelect<T>(props: AriaSelectOptions<T>, state: SelectState<T>,
         }
 
         if (props.onBlur) {
+          if (props.onFocusChange) {
+            props.onFocusChange(false);
+          }
```

Comment:
Same thing here as above, mind moving this if statement out of the wrapping `if (props.onBlur)`?

Reply:
fixed in the new commit, many thanks!